framework/console/controllers/BaseMigrateController.php

```diff
@@ -624,25 +727,42 @@ protected function migrateToVersion($version)
     protected function getNewMigrations()
     {
         $applied = [];
-        foreach ($this->getMigrationHistory(null) as $version => $time) {
-            $applied[substr($version, 1, 13)] = true;
+        foreach ($this->getMigrationHistory(null) as $class => $time) {
+            $applied[trim($class, '\\')] = true;
+        }
+
+        $migrationPaths = [];
+        if (!empty($this->migrationPath)) {
+            $migrationPaths[''] = $this->migrationPath;
```

Comment:
At the present state you can use both namespaced and not namespaced migrations at the same time.
It provides the support for the existing old projects.

We could mark `migrationPath` as deprecated, but this should be disscussed separately.

Composing migration paths at `init()` level is not necessary, I prefer to keep it this way at least for now.